_docs/quickstarts/enforcer/index.md

```diff
@@ -36,20 +36,39 @@ documentation.
 If a rule doesn't include a network name, then it's applied to all networks
 configured on the project. The network name is prepended to the rule name.
 
-Following is an example rule that allows SSH from anywhere:
+The following is an example rule that allows SSH from anywhere but blocks
```

Comment:
nit: Would it be clearer to say "allows SSH from anywhere and only internally allows tcp, udp, and icmp traffic"?

Reply:
@carise Excellent suggestion! Added!